nucypher/cli/painting.py

```diff
@@ -385,3 +393,77 @@ def paint_deployment_delay(emitter, delay: int = 3) -> None:
     for i in range(delay)[::-1]:
         emitter.echo(f"{i}...", color='yellow')
         time.sleep(1)
+
+
+def paint_stakers(emitter, stakers: List[str], agent) -> None:
+    current_period = agent.get_current_period()
+    emitter.echo(f"\nCurrent period: {current_period}")
+    emitter.echo("\n| Stakers |\n")
+    emitter.echo(f"{'Checksum address':42}  Staker information")
+    emitter.echo('=' * (42 + 2 + 53))
+
+    for staker in stakers:
+        nickname, pairs = nickname_from_seed(staker)
+        symbols = f"{pairs[0][1]}  {pairs[1][1]}"
+        emitter.echo(f"{staker}  {'Nickname:':10} {nickname} {symbols}")
+        tab = " " * len(staker)
+        info = agent.get_staker_info(staker)
+        stake, confirmed1, confirmed2, restake, *remaining_info = info
+        locked_restake_until_period, worker, worker_start, last_active = remaining_info
+
+        last_confirmed_period = max(confirmed1, confirmed2)
+        missing_confirmations = current_period - last_confirmed_period
+        stake_in_nu = round(NU.from_nunits(stake), 2)
+        locked_tokens = round(NU.from_nunits(agent.get_locked_tokens(staker)), 2)
+
+        emitter.echo(f"{tab}  {'Stake:':10} {stake_in_nu}  (Locked: {locked_tokens})")
+        emitter.echo(f"{tab}  {'Activity:':10} ", nl=False)
+        if missing_confirmations == -1:
+            emitter.echo(f"Next period confirmed (#{last_confirmed_period})", color='green')
+        elif missing_confirmations == 0:
+            emitter.echo(f"Current period confirmed (#{last_confirmed_period}). "
+                         f"Pending confirmation of next period.", color='yellow')
+        elif missing_confirmations == current_period:
+            emitter.echo(f"Never confirmed activity", color='red')
+        else:
+            emitter.echo(f"Missing {missing_confirmations} confirmations "
+                         f"(last time for period #{last_confirmed_period})", color='red')
+
+        emitter.echo(f"{tab}  {'Worker:':10} ", nl=False)
+        if worker == BlockchainInterface.NULL_ADDRESS:
+            emitter.echo(f"No worker set\n", color='red')
+        else:
+            emitter.echo(f"{worker}\n")
+
+
+def paint_locked_tokens_status(emitter, agent, periods) -> None:
+
+    MAX_ROWS = 30
+    period_range = list(range(1, periods + 1))
+    token_counter = Counter({day: agent.get_all_locked_tokens(day) for day in period_range})
```

Comment:
Just a reminder - this will print using only "active" workers/stakers

Reply:
How can we get the non active stakers too?

Reply:
It's not trivial question. Because if you want to print period by period all stakes for all stakers - this will be not fast, because should do this manualy (calculate next period, get next staker, get locked tokens, sum or get next staker, get all sub stakes and then do all others summarization)
Also possible other solution if only current/next periods will be enough

Reply:
Can this be derived by listing all the stakers that have ever connected to the network, then removing the currently active stakers?